src/utils/consts.rs

```diff
@@ -0,0 +1,3 @@
+pub const MOSAIC_TMP_LOG_FILE: &str = "/tmp/mosaic-log.txt";
+pub const MOSAIC_TMP_LOG_DIR: &str = "/tmp/mosaic-log";
+pub const MOSAIC_TMP_FOLDER: &str = "/tmp/mosaic";
```

Comment:
@5c077m4n A minor nitpick, but can the `mosaic-log` directory and `mosaic-log.txt` be moved inside `mosaic` folder?

Reply:
You are right, I just left it as it was so custom scripts won't break @henil